commit 9ba5cf37a41df263ae9c010670682cee37535513
Author: Christoph Moench-Tegeder <cmt@burggraben.net>

    lower gtk4 version dependency
    
    we don't have 4.22 yet, and 4.20 seems to work without major fallout

diff --git meson.build meson.build
index 8393facc..ef6409ee 100644
--- meson.build
+++ meson.build
@@ -77,7 +77,7 @@ endif
 version_h = vcs_tag(command: ['git', 'rev-parse', 'HEAD'], input: 'version.h.in', output: 'version.h', fallback: '')
 version = declare_dependency(sources : version_h, include_directories : include_directories('.'))
 
-gtk = dependency('gtk4', version : '>= 4.22')
+gtk = dependency('gtk4', version : '>= 4.20')
 gio = dependency('gio-2.0', version: '>= 2.88')
 gmodule = dependency('gmodule-2.0', version: '>= 2.88')
 gee = dependency('gee-0.8', version: '>= 0.8.5')
